actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -480,6 +487,8 @@ def []=(name, options)
           options = { value: options }
         end
 
+        preprocess_options(options)
```

Comment:
@mikeycgto :smile: 
Correct me if I'm wrong:
We now want `:expires` to support duration, i.e. we want to specify the duration for which the cookie is valid like this:
`cookies.signed[:user_name] = { value: "bob", expires: 2.hours }`
And since we're tacking on the value of `options[:expires] ` to `:expires_at` in case of signed / encrypted cookies, isn't it necessary to do `preprocess_options` before commit method is called, since `@parent_jar[name] = options` is only called later on.